src/tests/ftest/osa/osa_offline_parallel_test.py

```diff
@@ -79,27 +97,41 @@ def run_offline_parallel_test(self, num_pool, data=False):
         # Exclude rank 2.
         rank = 2
 
+        test_seq = self.ior_test_sequence[0]
         for val in range(0, num_pool):
             pool[val] = TestPool(self.context,
                                  dmg_command=self.get_dmg_command())
             pool[val].get_params(self)
-            # Split total SCM and NVME size for creating multiple pools.
-            pool[val].scm_size.value = int(pool[val].scm_size.value /
-                                           num_pool)
-            pool[val].nvme_size.value = int(pool[val].nvme_size.value /
-                                            num_pool)
             pool[val].create()
             pool_uuid.append(pool[val].uuid)
             self.pool = pool[val]
             if data:
-                self.write_single_object()
+                self.run_ior_thread("Write", oclass, test_seq)
+                if oclass != "S1":
+                    self.run_mdtest_thread()
+                if self.test_during_aggregation is True:
+                    self.run_ior_thread("Write", oclass, test_seq)
```

Comment:
No. This will not start aggregation... If test.during_aggregation test is run, we create two containers within the pool and destroy one container. I will some code comments to explain the issue. The Aggregation is started at:
![image](https://user-images.githubusercontent.com/48604608/114642455-f2aca300-9ca1-11eb-8925-c36d77aeb06c.png)

Reply:
Discussed with Samir and provided the details. I have added comments to the code.